Src/ILGPU/Backends/OpenCL/CLCodeGenerator.cs

```diff
@@ -306,7 +309,9 @@ internal CLCodeGenerator(in GeneratorArgs args, Method method, Allocas allocas)
         /// </summary>
         public IntrinsicImplementationProvider<CLIntrinsic.Handler>
             ImplementationProvider
-        { get; }
+        {
```

Comment:
ISSUE: Keep as `{ get; }`.

Reply:
@MoFtZ I think that this is specific  `{ get; }` since I tried various combinations and non seems to work for this case
but when I look at others, it works as expected
the only thing different is that this `{ get; }` is on a new line... I would like to suggest the following changes here:
original 
```
            ImplementationProvider
        { get; }
```

after formating
```
            ImplementationProvider { get; }
```
is this ok from a code perspective (functionality, readability, etc..?)
from format code perspective, this will not be changed when `code-style` job is triggered

Reply:
Ah, I see the problem now.

The full line should be, but it does not fit in the 90 char limit:
```CSharp
public IntrinsicImplementationProvider<CLIntrinsic.Handler> ImplementationProvider { get; }
```
So, as you suggested, we changed it to:
```CSharp
public IntrinsicImplementationProvider<CLIntrinsic.Handler>
    ImplementationProvider { get; }
```

However, in VS2019, when we apply the `Format Document` command, it further transforms the text to:
```CSharp
public IntrinsicImplementationProvider<CLIntrinsic.Handler>
    ImplementationProvider
{ get; }
```

@ljubon Your suggestion to use `ImplementationProvider { get; }` sounds like a good idea. We will occasionally fight with VS2019, but then the format script will re-apply the desired style.

Reply:
@MoFtZ thank you for checking this further! I didn't notice 90 char limit, but it makes sense at the end!
Will push new change in next commit 🚀